core/controllers/download.py

```diff
@@ -43,7 +43,12 @@ def download():
         itl_regions,
         outcome_categories,
     )
-    data_generator = serialise_download_data(base_query, outcome_categories)
+
+    if outcome_categories:
```

Comment:
I wonder, rather than have lots of if/else's around with different params in namespace, could we just rename the original `base_query` var as `query`, then we don't have `base_query` and `query` in namespace, and we don't need the else block to re-assign one to the other. i.e. only extend if we need to and overwrite the original var?

Reply:
Good idea - updated in a later commit in this PR